src/Symfony/Component/Security/Core/User/MissingUserProvider.php

```diff
@@ -18,6 +18,10 @@
  * when a firewall requires a user provider but none was defined.
  *
  * @internal
+ *
+ * @template TUser of UserInterface
+ *
+ * @template-implements UserProviderInterface<TUser>
```

Comment:
This class should not be generic, and use `UserProviderInterface<UserInterface>`

Reply:
Done